3-nodes-mirror-3-dc/files/config.yaml

```diff
@@ -52,7 +52,7 @@ domains_config:
       nto_select: 3
     ssid: 1
   security_config:
-    enforce_user_token_requirement: false      # User 'root' with the empty password out of box
+    enforce_user_token_requirement: true      # User 'root' with the empty password out of box
```

Comment:
It's not really related, but IIRC, the initial setup playbook configures some non-empty password for root by default, so this comment looks outdated.

Reply:
Comment is removed